src/sbt-test/sbt-scraml/cats/test

```diff
@@ -0,0 +1,5 @@
+> compile
+# compile again, should not yield errors
```

Comment:
I think it's enough to check the "double compilation" test only once in the `simple` test. It's mainly for verifying the cleanup and caching. Kudos for having meaningful assertions :tada:

Reply:
Gotcha.  I'll remove the double compilation from `cats` and merge the branch 👍 .